client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -330,24 +391,38 @@ class DomainDetailsForm extends Component {
 		);
 	}
 
+	renderExtraDetailsForm = () => {
+		return ( <ExtraInfoFrForm
+			isProbablyOrganization={ Boolean( formState.getFieldValue( this.state.form, 'organization' ) ) }
+			values={ this.state.registrantExtraInfo }
+			onStateChange={ this.handleExtraChange } >
+				{ this.renderSubmitButton() }
+			</ExtraInfoFrForm>
+		);
+	}
+
 	handleCheckboxChange = () => {
 		this.setPrivacyProtectionSubscriptions( ! this.allDomainRegistrationsHavePrivacy() );
 	}
 
-	closeDialog = () => {
-		this.setState( { isDialogVisible: false } );
+	closeDialog( dialogKey = 'isDialogVisible' ) {
```

Comment:
these _might_ need to be bound to the instance like they were before, since they're using `this`. just depends on how they're called..